msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -38,7 +38,7 @@ namespace Microsoft.Identity.Client
     /// <summary>
     /// Class to be used for confidential client applications like Web Apps/API.
     /// </summary>
-    public sealed class ConfidentialClientApplication : ClientApplicationBase, IConfidentialClientApplication
+    public sealed class ConfidentialClientApplication : ClientApplicationBase, IConfidentialClientApplication, IConfidentialClientApplicationWithCertificate
```

Comment:
>IConfidentialClientApplicationWithCertificate [](start = 111, length = 45)

It seems like the obo flow is missing for MSAL, or am I missing something?

Reply:
I agree. It's in ADAL.

Reply:
#Resolved